### ISSUE TITLE
'ascii' codec can't encode character u'\u270a'

### ISSUE THREAD

**Issue by tibor** [InstaPy/InstaPy#160]
Seems like something is wrong with the emoji encoding:

```
print('--> Commented: {}'.format(rand_comment))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u270a' in position 31: ordinal not in range(128)
```

(I'm on the latest version, scripts doesn't work with any emoji which previously worked)

**Comment by eduardocalazansjr** [comment]
Can't reproduce, maybe it's not OS locale related, since the code now uses string.format?
Check your system locale and give us a feedback.

![screen shot 2017-04-19 at 14 21 24](https://cloud.githubusercontent.com/assets/5658870/25192958/995a07e6-250b-11e7-9b24-2086130af69b.png)

**Comment by eduardocalazansjr** [comment]
Tested it on OSx and seems to work, but on linux it doesn't.
Maybe another thing that goes live without testing, clearly is missing 'u' before the string on output. 

Check https://github.com/timgrossmann/InstaPy/pull/161

**Comment by tibor** [comment]
Can't confirm this working on Mac as well.

For future issues: which information should be at least included in a issue (python version, version number, OS, anything else?)?

Maybe we should add this information to README.md as well to make bugtracking easier.

**Comment by eduardocalazansjr** [comment]
It's not python version related, maybe console output related (since OSx terminal is kinda of relaxed with this), maybe just OS is relevant.

Seems to be fixed on #161, will track another part of code that need this.

**Comment by pekapa** [comment]
Might be python version related since one of the major 2to3 differences is how strings and unicodes are treated.

We should create a template to help people opening issues.

**Comment by tibor** [comment]
I'm using Python 2.7.10 here as well.

**Comment by eduardocalazansjr** [comment]
Still on the last revision of string documentation, not version related since there's no unicode flag or prefix on code and 3.5+ still support string literal prefixes.
Just an lack of testing before going live.

https://www.python.org/dev/peps/pep-0414/

@tibor could you confirm that the fix proposed on #161 fixes this issue? I'm testing in some droplets and doesn't seems to be any problem with emoji output.

**Comment by tibor** [comment]
Works for me, Thanks!
